src/app.js

```diff
@@ -59,7 +59,11 @@ function listen (koaApp, config, ledgers) {
   // Start a coroutine that connects to the backend and
   // subscribes to all the ledgers in the background
   co(function * () {
-    yield backend.connect()
+    const success = yield backend.connect()
+    if (!success) {
+      log('app').error('backend initialization has failed - the connector will now exit')
+      process.exit(1)
```

Comment:
You can change the code in the backend to handle that. Specifically, [this part](https://github.com/interledger/five-bells-connector/pull/173/files/26409ef3c1ea9d91d82f9340ef6b051d0e46fe68#diff-179f9c6e4bfdb0a397cc134cd45e1b48R46)